guardian/floodgate
# Bump node to latest, and pin yarn version

## What does this change?

Bump node to latest, and pin yarn version, in the root of the project.

## How to test

- Build commands should work as before
- Deploy to CODE. Frontend should work as expected

Tested in CODE ✅
